packages/manager/apps/cloud/webpack.config.js

```diff
@@ -32,75 +28,47 @@ function readNgAppInjections(file) {
   return injections;
 }
 
-function getNgAppInjections(region) {
-  const injections = [
-    ...readNgAppInjections(`./.extras-${region}/ng-app-injections`),
-    ...readNgAppInjections('./.extras/ng-app-injections'),
-  ];
-
-  const ngAppInjections = injections.map((val) => `'${val}'`).join(',');
+function getNgAppInjections(regions) {
+  return regions.reduce((ngAppInjections, region) => {
+    const injections = [
+      ...readNgAppInjections(`./.extras-${region}/ng-app-injections`),
+      ...readNgAppInjections('./.extras/ng-app-injections'),
+    ];
 
-  return ngAppInjections || 'null';
+    return {
+      ...ngAppInjections,
+      [region]: JSON.stringify(injections),
+    };
+  }, {});
 }
 
-fs.readdirSync(folder).forEach((file) => {
-  // skip config folder, it'll be added later depending on current environment
-  if (file === 'config') {
-    return;
-  }
-  const stats = fs.lstatSync(`${folder}/${file}`);
-  if (stats.isDirectory()) {
-    const jsFiles = glob.sync(`${folder}/${file}/**/!(*.spec|*.mock).js`);
-    if (jsFiles.length > 0) {
-      bundles[file] = jsFiles;
-    }
-  }
-});
-
 module.exports = (env = {}) => {
-  const REGION = _.upperCase(env.region || process.env.REGION || 'EU');
-
-  const { config } = webpackConfig(
-    {
-      template: './client/index.html',
-      basePath: './client',
-      lessPath: ['./client/app', './client/components', './node_modules'],
-      lessJavascriptEnabled: true,
-      root: path.resolve(__dirname, './client/app'),
-      assets: {
-        files: [
-          { from: path.resolve(__dirname, './client/assets'), to: 'assets' },
-          { from: foundNodeModulesFolder('angular-i18n'), to: 'angular-i18n' },
-          {
-            from: path.resolve(__dirname, './client/**/*.html'),
-            context: 'client',
-          },
-        ],
-      },
+  const { config } = webpackConfig({
+    template: './client/index.html',
+    basePath: './client',
+    lessPath: ['./client/app', './client/components', './node_modules'],
+    lessJavascriptEnabled: true,
+    root: path.resolve(__dirname, './client/app'),
+    assets: {
+      files: [
+        { from: path.resolve(__dirname, './client/assets'), to: 'assets' },
+        { from: foundNodeModulesFolder('angular-i18n'), to: 'angular-i18n' },
+        {
+          from: path.resolve(__dirname, './client/**/*.html'),
+          context: 'client',
+        },
+      ],
     },
-    REGION ? Object.assign(env, { region: REGION }) : env,
```

Comment:
We noticed that `env` is now missing from the shared configuration.
Here is the patch associated with: #3427